node/resources/brand.ts

```diff
@@ -0,0 +1,26 @@
+import { pick } from 'ramda'
+import { createXLSX, jsonToXLSXFields } from './xlsx'
+
+enum BrandTranslatableField {
+  metaTagDescription = 'metaTagDescription',
+  name = 'name',
+  title = 'title',
+}
+
+const getBrandTranslatableFields = pick(Object.values(BrandTranslatableField))
+
+type BrandTranslatables = Record<BrandTranslatableField, string>
+
+const jsonToXLSXField: BrandTranslatables = {
+  metaTagDescription: '_MetaTagDescription',
+  name: '_Name',
+  title: '_Title',
+}
+
+export async function getBrandXLSX(ctx: Context) {
+  const { clients: { catalog } } = ctx
+  const brands = await catalog.exportBrands() as Array<Record<string, string>>
```

Comment:
⛏ In my opinion, it'd be better to type the `exportBrands` method instead of having to typecast its result whenever it's used.